### DISCUSSION TITLE
webrtc do not work?

### DISCUSSION THREAD

**Discussion by MarkStephan89** [Ylianst/MeshCentral#5965] [General]
hi, 
I use mc 1.1.22. 
If I transfer files over mc-browser and a agent the mc-dockercontainer have a higher network usage (and a bit CPU). 

![image](https://github.com/Ylianst/MeshCentral/assets/68630782/0a31ad6c-3f20-4132-870d-fcdd1ea79c1c)

Is this normal?

In my opinion the browser an the agent should connect direct together?

Here my config: 

```
{
  "$schema":"http://info.meshcentral.com/downloads/meshcentral-config-schema.json",
  "settings":{
    "cert":"XXX",
    "IgnoreAgentHashCheck":true,
    "WsCompression":true,
    "AgentWsCompression":true,
    "Compression":true,
    "sessionKey":"XXX",
    "BrowserPing":50,
    "BrowserPong":50,
    "AgentPing":50,
    "AgentPong":50,
    "WebRTC":"true",
    "AutoBackup":{
      "backupPath":"/opt/meshcentral/meshcentral-backups",
      "backupInvervalHours":24,
      "keepLastDaysBackup":"",
      "zippassword":""
    }
  },
  "domains":{
    "":{
      "minify":true,
      "NewAccounts":"false",
      "certUrl":"https://XXX:443",
      "_SessionRecording":{
        "filepath":"/opt/meshcentral/mescentral-recording",
        "index":true,
        "maxRecordingSizeMegabytes":500,
        "protocols":[1,2,200]
      }
    }
  }
}
```

Or have I thought incorrectly about that?

Many Thanks!

**Comment by silversword411** [comment]
Where are those graphs from?

**Reply by si458** [reply]
@silversword411 portainer (docker)

**Comment by si458** [comment]
@MarkStephan89 yes this is normal to see memory and cpu usage when you are connecting and file transferring etc
as far as im aware, the file transfer doesnt work over webrtc, webrtc is only for the remote control

**Reply by MarkStephan89** [reply]
ah ok thanks!
